src/api/config/base-params.md

```diff
@@ -0,0 +1,23 @@
+# Base Parameters
```

Comment:
Are there any requirements for the key?

* Could I introduce a new peer with random keys (given a proper algorithm that generates them and that they have a proper format)?
* Could I randomly change keys for a peer at some moment, restart it and expect it to work?

Reply:
1. Well, each peer might have any keys you want it to use. I lack understanding of the bigger picture and how keys should be approached in real-world scenarios.
2. Not sure, don't think so.